vmngclient/api/template_api.py

```diff
@@ -317,6 +318,35 @@ def _delete_cli_template(self, name: str) -> bool:
             raise AttachedError(template.name)
         return True
 
+    @overload
+    def edit(self, template: FeatureTemplate) -> Any:
+        ...
+
+    @overload
+    def edit(self, template: CLITemplate) -> Any:
+        pass
```

Comment:
use consistent style for overload body (... instead pass is used in whole repository)